timezone/init.sls

```diff
@@ -1,26 +1,27 @@
-# This state configures the timezone.
+# -*- coding: utf-8 -*-
+# vim: ft=sls
 
-{%- set timezone = salt['pillar.get']('timezone:name', 'Europe/Berlin') %}
-{%- set utc = salt['pillar.get']('timezone:utc', True) %}
-{% from "timezone/map.jinja" import confmap with context %}
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/map.jinja" import timezone with context %}
 
 timezone_setting:
   timezone.system:
-    - name: {{ timezone }}
-    - utc: {{ utc }}
+    - name: {{ timezone.name }}
+    - utc: {{ timezone.utc }}
 
-  {%- if grains.os not in ('MacOS', 'Windows') %}
+{%- if grains.os not in ('MacOS', 'Windows') %}
 
 timezone_packages:
   pkg.installed:
-    - name: {{ confmap.pkgname }}
+    - name: {{ timezone.pkgname }}
```

Comment:
This should be `timezone.pkg.name` because `pkg` will be a dict in template-formula.